bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -641,5 +657,29 @@ pub mod pallet {
 			let signing_root = Self::compute_signing_root(header, domain)?;
 			Ok(signing_root)
 		}
+
+		/// Updates are free if the update is successful and the interval between the latest
+		/// finalized header in storage and the newly imported header is large enough. All
+		/// successful sync committee updates are free.
+		pub(super) fn may_refund_call_fee(
+			latest_slot: u64,
+			improved_by_slot: u64,
+			sync_committee_updated: bool,
+		) -> Pays {
+			// If the sync committee was successfully updated, the update may be free.
+			if sync_committee_updated {
+				return Pays::No;
+			}
```

Comment:
This effectively is just returning the parameter of `sync_committee_updated` of mapped to `Pays`, which seems redundant.


And given my other comment above, I would advise restructuring as follows:

```
fn check_refundable(update: &Update, latest_slot: u64) -> Pays
```